safe/messaging/item/message_element.py

```diff
@@ -69,6 +70,7 @@ def _is_qstring(message):
     def _is_stringable(message):
         return (isinstance(message, basestring)
                 or isinstance(message, int)
+                or isinstance(message, str)
```

Comment:
str is implemented from basestring class. No need to have this as it will be catched by isinstance(message, basestring) above